java/source/io.rst

```diff
@@ -457,18 +504,19 @@ Read - From File
     ){
         // write
         writer.start();
-        for (int i=0; i<10; i++){
+        for (int i=0; i<2; i++){
             // Generate data or modify the root or use a VectorLoader to get fresh data from somewhere else
             writer.writeBatch();
         }
 
         // read
         try (FileInputStream fileInputStreamForStream = new FileInputStream(file);
              ArrowStreamReader reader = new ArrowStreamReader(fileInputStreamForStream, rootAllocator)){
-            // read the batch
-            reader.loadNextBatch();
-            VectorSchemaRoot vectorSchemaRootRecover = reader.getVectorSchemaRoot();
-            System.out.print(vectorSchemaRootRecover.contentToTSVString());
+            while(reader.loadNextBatch()){
+                // read the batch (on the next example you could see how to use a VectorLoader to get fresh data)
```

Comment:
I don't think this is relevant to this recipe? It really sounds like we should have more recipes for loader/unloader.

Reply:
Deleted and added a separate recipe to demonstrate VectorLoader/VectorUnloader.